case-lib/pipeline.sh

```diff
@@ -48,5 +49,6 @@ func_pipeline_parse_value()
     local idx=$1
     local key=$2
     [[ $idx -ge $PIPELINE_COUNT ]] && echo "" && return
-    eval echo "\${PIPELINE_$idx['$key']}"
+    # shellcheck disable=SC1087
+    eval echo "\${PIPELINE_$idx['$key']}" # dynmaic echo the target value of the PIPELINE
```

Comment:
Using an intermediate variable (with a good name) should get rid of SC1087 while having simpler quoting.